### PR TITLE
Digest moved to art map

### PR DESCRIPTION
**Pull request by genaforvena**

### CODE REVIEW

**Comment by genaforvena** [comment]
@sanovikov71 @oleg-popenov

File: app/src/main/java/org/imozerov/streetartview/ui/explore/map/ArtMapFragment.kt [hunk 1]
```diff
@@ -2,43 +2,52 @@ package org.imozerov.streetartview.ui.explore.map
...
 import org.imozerov.streetartview.ui.explore.interfaces.ArtView
 import org.imozerov.streetartview.ui.explore.interfaces.Filterable
 import org.imozerov.streetartview.ui.extensions.addArtObject
 import org.imozerov.streetartview.ui.extensions.addUserLocationMarker
 import org.imozerov.streetartview.ui.extensions.getCurrentLocation
+import org.imozerov.streetartview.ui.extensions.loadImage
 import org.imozerov.streetartview.ui.model.ArtObjectUi
 
 class ArtMapFragment : Fragment(), Filterable, ArtView {
+    val TAG = "ArtMapFragment"
```
Comment (sanovikov71) [inline app/src/main/java/org/imozerov/streetartview/ui/explore/map/ArtMapFragment.kt:31]: Looks strange

Comment (sanovikov71) [inline app/src/main/java/org/imozerov/streetartview/ui/explore/map/ArtMapFragment.kt:31]: oh it's for logs and the other for fragmentmanager, ok

File: app/src/main/java/org/imozerov/streetartview/ui/extensions/UiExtensions.kt [hunk 2]
```diff
@@ -14,29 +11,30 @@ val TAG = "UiExtensions"
...
-                                    }
-
-                                    override fun onError() {
-                                        Log.w(TAG, "Could not fetch image")
-                                    }
-                                })
-                    }
-                })
+                .into(this)
+        //                .networkPolicy(NetworkPolicy.OFFLINE)
```
Comment (sanovikov71) [inline app/src/main/java/org/imozerov/streetartview/ui/extensions/UiExtensions.kt:17]: Maybe it should be removed

**Pull request merged**